.github/workflows/pr-go.yaml

```diff
@@ -70,10 +70,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
```

Comment:
Not sure why it worked until now.
We need this so it can find the binary in the go/bin path.

Error: `protolock: No such file or directory`